docs/README.md

```diff
@@ -119,10 +119,12 @@ The compiled binary will be built to `./target/release/logdna-agent`.
 
 ### Options
 
-The agent accepts configuration from two sources, environment variables and a configuration YAML file. The default configuration yaml file is located at `/etc/logdna/config.yaml`. The following options are available:
+The agent accepts configuration from three different sources: environment variables, command line arguments and/or a
+configuration YAML file. The default configuration yaml file is located at `/etc/logdna/config.yaml`. The following
```

Comment:
The config yaml file usage still has rough edges: I've filed LOG-9482, but there are more usability issues.